server/test/serverTest.js

```diff
@@ -1,575 +1,661 @@
-
-            response = await chai.request(server).post('/alert/sms').send(twilioMessageUnit1_IncidentNotesResponse)
-            expect(response).to.have.status(200)
- 
-            sessions = await db.getAllSessionsWithButtonId(unit1UUID)
-            expect(sessions.length).to.equal(1)
-            expect(sessions[0].state, 'state after staff have provided incident notes').to.deep.equal(ALERT_STATE.COMPLETED) 
-
-            // now start a new session for a different unit
-            // Power Automate always sends two requests
-            await chai.request(server).post('/').send(unit2FlicRequest_SingleClick)
-            await chai.request(server).post('/').send(unit2FlicRequest_SingleClick)
-
-            sessions = await db.getAllSessionsWithButtonId(unit2UUID)
-            expect(sessions.length).to.equal(1)
-            expect(sessions[0].state, 'state after new button press from a different unit').to.deep.equal(ALERT_STATE.STARTED)
-            expect(sessions[0].buttonId).to.deep.equal(unit2UUID)
-            expect(sessions[0].unit).to.deep.equal('2')
-            expect(sessions[0].numPresses).to.deep.equal(1)
-        });
-
-        it('should send a message to the fallback phone number if enough time has passed without a response', async () => {
-            // Power Automate always sends two requests
-            let response = await chai.request(server).post('/').send(unit1FlicRequest_SingleClick)
-            response = await chai.request(server).post('/').send(unit1FlicRequest_SingleClick)
-
-            expect(response).to.have.status(200)
-            await sleep(4000)
-            let sessions = await db.getAllSessionsWithButtonId(unit1UUID)
-            expect(sessions.length).to.equal(1)
-            expect(sessions[0].state, 'state after reminder timeout has elapsed').to.deep.equal(ALERT_STATE.WAITING_FOR_REPLY);
-            expect(sessions[0].fallBackAlertTwilioStatus).to.not.be.null
-            expect(sessions[0].fallBackAlertTwilioStatus).to.not.equal('failed')
-            expect(sessions[0].fallBackAlertTwilioStatus).to.not.equal('undelivered')
-            expect(sessions[0].fallBackAlertTwilioStatus).to.be.oneOf(['queued', 'sent', 'delivered'])
-
-        });
-    });
-});
+      expect(imports.braveAlerter.startAlertSession).to.be.calledOnce
+
+      expect(imports.braveAlerter.sendSingleAlert).to.be.calledWith(
+        sinon.match.any,
+        sinon.match.any,
+        'This in an urgent request. The button has been pressed 2 times. Please respond "Ok" when you have followed up on the call.',
+      )
+    })
+
+    it('should send the initial text message after a valid single click request to /flic_button_press', async () => {
+      await chai.request(server).post('/flic_button_press').set('button-serial-number', unit1SerialNumber).set('button-battery-level', '100').send()
```

Comment:
Hmmm, this is an ugly turn of events. Oh well.

Reply:
I'm not sure I am seeing what you're talking about, is it long lines or?

Reply:
I just meant because it used to be clear what was going on with the dots lined up, but now it's all on one line. Oh well.

Reply:
ohhh yeah that is unfortunate :(

Reply:
I'm looking at this now and I'm really unclear on why it is making this all one line and leaving the next one on separate lines??? I put it back to be nice looking for now with a disable but can undo that depending on how you feel about it

Reply:
Nah, I'd prefer without the `disable`. Thanks though!